File: Android/dokit-weex/src/main/java/com/didichuxing/doraemonkit/weex/storage/StorageInfo.java
package com.didichuxing.doraemonkit.weex.storage;

import java.io.Serializable;

/**
 * @author haojianglong
 * @date 2019-06-18
 */
public class StorageInfo implements Serializable {

    public String key;
    public String value;
    public String timestamp;

    public StorageInfo() {
    }

    public StorageInfo(String key, String value) {
        this.key = key;
        this.value = value;
    }

}

File: Android/kotlin/doraemonkit-weex/src/main/java/com/didichuxing/doraemonkit/weex/storage/StorageInfo.kt
package com.didichuxing.doraemonkit.weex.storage

import java.io.Serializable

/**
 * Transformed by alvince on 2020/7/1
 *
 * @author haojianglong
 * @date 2019-06-18
 */
class StorageInfo(
    var key: String = "",
    var value: String = ""
) : Serializable {

    var timestamp: String? = null

}